tests/integration/test_searchsorted.py

```diff
@@ -0,0 +1,154 @@
+# Copyright 2022 NVIDIA Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import numpy as np
+import pytest
+
+import cunumeric as num
+
+
+def compare_assert(a_np, a_num):
+    if not num.allclose(a_np, a_num):
```

Comment:
a nitpick: can't this be `array_equal`? I think `searchsorted` shouldn't ever do anything that could cause rounding errors